dora/core/client/fs/src/main/java/alluxio/client/block/stream/DataWriter.java

```diff
@@ -57,9 +57,6 @@ private Factory() {} // prevent instantiation
     public static DataWriter create(FileSystemContext context, long blockId, long blockSize,
         WorkerNetAddress address, OutStreamOptions options) throws IOException {
       AlluxioConfiguration alluxioConf = context.getClusterConf();
-      boolean shortCircuit = alluxioConf.getBoolean(PropertyKey.USER_SHORT_CIRCUIT_ENABLED);
```

Comment:
In `GrpcDataWriter` there are comments mentioning "short-circuit". I found those occurrences by searching "shortcircuit" and "short-circuit" and "short circuit".

Reply:
Some other occurrences include `Format.java`, `package-info.java`, `BlockInStream.java`, `DecommissionWorkerCommand` etc.

Reply:
all removed